.github/pull_request_template.md

```diff
@@ -20,14 +23,16 @@ See Story: [FSA22V2-ISSUE](https://sparkbox.atlassian.net/browse/FSA22V2-ISSUE)
 3. In terminal: `npm run dev`.
 4. Check [localhost:3000](http://localhost:3000/)for changes.
```

Comment:
This is a minor nitpick that I noticed on previous PRs, but could you add a space after `(http://localhost:3000/)`?